draft-thomson-httpbis-http2bis.xml

```diff
@@ -1167,231 +1162,77 @@
           </t>
         </section>
       </section>
+
       <section anchor="StreamPriority">
-        <name>Stream Priority</name>
-        <t>
-          A client can assign a priority for a new stream by including prioritization information in
-          the <xref target="HEADERS">HEADERS frame</xref> that opens the stream.  At any other time,
-          the <xref target="PRIORITY">PRIORITY frame</xref> can be used to change the priority of a
-          stream.
-        </t>
-        <t>
-          The purpose of prioritization is to allow an endpoint to express how it would prefer its
-          peer to allocate resources when managing concurrent streams.  Most importantly, priority can
-          be used to select streams for transmitting frames when there is limited capacity for
-          sending.
-        </t>
+        <name>Prioritization</name>
         <t>
-          Streams can be prioritized by marking them as dependent on the completion of other streams
-          (<xref target="pri-depend"/>).  Each dependency is assigned a relative weight, a number
-          that is used to determine the relative proportion of available resources that are assigned
-          to streams dependent on the same stream.
+          In a multiplexed protocol like HTTP/2, prioritizing allocation of resources to streams can
```

Comment:
This is imo somewhat unclear language: what exactly are the "resources" here? When talking about web page construction in other contexts, a "resource" is often a file (say main.js), while I feel here resources is more akin to bandwidth (as files are implicitly linked to individual streams?)